File: output/ChatGPT/codenet/Python/C++/s174910557.cpp
#include <iostream>
using namespace std;

int main() {
    int N;
    cin >> N;
    string S;
    cin >> S;

    int ans = 1;
    char tmp = S[0];
    for (char s : S) {
        if (s != tmp) {
            ans += 1;
            tmp = s;
        }
    }

    cout << ans << endl;

    return 0;
}

File: output/ChatGPT/codenet/Python/Java/s174910557.java
import java.util.Scanner;

public class s174910557 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int N = scanner.nextInt();
        String S = scanner.next();

        int ans = 1;
        char tmp = S.charAt(0);
        for (int i = 1; i < N; i++) {
            char s = S.charAt(i);
            if (s != tmp) {
                ans++;
                tmp = s;
            }
        }

        System.out.println(ans);
    }
}